docs/tasks/retiring-a-smart-answer.md

```diff
@@ -35,26 +35,25 @@ bundle exec rake "publishing_api:unpublish_redirect[d0a95767-f6ab-432a-aebc-096e
 
 If there is not content to replace the Smart Answer then the convention is to
 serve a page that indicates there used to be content but isn't anymore (a 410
-gone HTTP response). An example of how to perform this is:
+gone HTTP response). To do this you'd run the following command:
 
 ```
 bundle exec rake publishing_api:unpublish_gone[<content_id>]
 ```
 
-> It is possible to remove the Smart Answer and rather than serve a 410 Gone
-> page instead serve a 404 Not Found, which misleads visitors to the page that
-> content had never been published there. This can be done by using the
-> `publishing_api:unpublish_vanish` task instead of the gone equivalent. This
-> task should only be used in exceptional circumstances for example an
-> accidental early publishing of content with a sensitive URL.
+> It is possible to remove the Smart Answer and serve a 404 Not Found response (instead of a
+> 410 Gone response) which misleading implies that content had never been published there.
+> This can be done by using the `publishing_api:unpublish_vanish` task instead of the gone
+> equivalent. This task should only be used in exceptional circumstances for example accidental
+> early publishing of content with a sensitive URL.
 
 ### Replace a Smart Answer with a different content type
 
 Sometimes there will be a need to replace a Smart Answer with a different type
-of content, typically these have been transactions, answers and start pages
-that can be published through [Publisher](https://github.com/alphagov/publisher).
+of content, typically these have been transactions, answers or start pages
+which are published through [Publisher](https://github.com/alphagov/publisher).
 
-The next step is to perform a temporary removal of the Smart Answer page(s) to
+You will need to perform a temporary removal of the Smart Answer page(s) to
```

Comment:
Maybe we should add something like "Before a new piece of content can be recreated with the same URL" or similar to reference the need to do this first?